libraries/config/include/config/config.hpp

```diff
@@ -43,6 +45,8 @@ struct NetworkConfig {
   uint16_t network_performance_log_interval = 0;
   uint16_t network_num_threads = std::max(uint(1), uint(std::thread::hardware_concurrency() / 2));
   uint16_t network_packets_processing_threads = 10;
+  uint16_t network_peer_blacklist_timeout = kBlacklistTimeoutDefaultInSeconds;
```

Comment:
maybe we can reuse `network_peer_blacklist_timeout` with value of `0` instead of new param `disable_peer_blacklist`

Reply:
I remember now that when I originally thought about this I wanted to use 0 for infinite timeout although I guess I did not implement it that way. Do you think it is better to have 0 as disable or having 0 as infinite timeout?

Reply:
but then this map can be huge, if we never clean it up, maybe there should be some hard limit like 24h or something